engine/Model.php

```diff
@@ -30,6 +30,22 @@ public function __construct($current_module = NULL) {
             echo $this->error; die();
         }
 
+        /*
+           Make debuging mysql accesible from the config file:
+           Replace 'private $debug = false;' with 'private $debug = MODELDEBUG;' (line 12) 
+
+           In /config/config.php add:
+           define('MODELDEBUG', 'true');
+           If the above line is not found, the debug setting will default to false
+
+           Then, add the below 5 lines
+        */
+        if (MYSQLDEBUG !== ''|| MYSQLDEBUG !== 'true'){ 
```

Comment:
You can put this in a one liner something like this
`defined(MYSQLDEBUG) ?: define('MODELDEBUG', false);`